### PR TITLE
Update Displays list styling

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Update Displays list styling

Use madero-style
Update header, search box, add button, error boxes
Update spacing for list as well as details

Show white background

[stage-19]

## Motivation and Context
Network Improvements epic.

## How Has This Been Tested?
Tested changes locally.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
Initial PR. Could be merged, as there are no major issues.

@ezequielc please review. Thanks!

File: web/scripts/app.js [hunk 1]
```diff
@@ -170,6 +170,7 @@ angular.module('risevision.apps', [
       $rootScope.$on('$stateChangeSuccess', function (event, toState) {
         if (toState.name === 'apps.schedules.details' ||
           toState.name === 'apps.schedules.add' ||
+          toState.name === 'apps.displays.list' ||
```
Comment (ezequielc) [inline web/scripts/app.js:173]: @alex-deaconu At the end of this epic, we can probably get rid of this white background hack. Thanks!

**Review by ezequielc — APPROVED** [review]
@alex-deaconu Looks great 👍 
I am wondering if we should stack Search/Add Display on mobile. 
Also, do you remember if we should keep Weekly Templates in Displays List or remove it? Thanks!

File: web/scripts/app.js (hunk 1 above)
Comment (alex-deaconu) [reply web/scripts/app.js:173]: @ezequielc good point, I will make a note of doing it after updating the other 2 lists. Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc re: stacking, I took a look at that but didn't find an easy way to achieve it with the current styling. I think we can do it as a secondary task as it will apply to all the lists.

Re: Weekly Templates, I don't know, I'm not aware of any decision to leave/remove.

Thanks.

**Comment by alex-deaconu** [comment]
Force pushed because I had forgotten the stage tag on the merge.

**Comment by alex-deaconu** [comment]
@ezequielc missed the require role directive for the Add Display button. Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu Good catch 👍

**Pull request merged**